docs/eventing/sources/sinkbinding/getting-started.md

```diff
@@ -1,6 +1,4 @@
-# Create a SinkBinding object
-
-![API version v1](https://img.shields.io/badge/API_Version-v1-red?style=flat-square)
```

Comment:
I'd keep this version flag since these are intentionally added by engineering folks

Reply:
I removed it because this flag is already on the overview page (README.md) for the SinkBinding section. Do you think it should be at the top of each page of the SinkBinding section, i.e. on the README, Creating a SinkBinding object, and reference page? I think it's weird to just have it on two of the three pages.

Reply:
Ah OK makes sense. My concern as per for other things in the past is that if a person doesn't go to the landing page and somehow ends up on one of the other SinkBinding pages it might be better to include it on all of them? But it's up to you.

Reply:
Sure I'll do that. Your concern makes sense. :)